Sage/carbon
# feat(checkbox): describe checkbox component using design tokens - FE-4375

### Proposed behaviour

Describes the checkbox component using design tokens.
Adds missing props in the checkbox.style.js file.
Updates tests that are related to the changes made.

### Current behaviour

The checkbox component uses the theme styled-component property.
The propTypes in the css files are missing the props used.

### Checklist




#### QA


### Additional context



### Testing instructions

The correct appearance can be checked in the component: Checkbox.
For theme: mint and aegean there should be no differences.
Use devtools to check if checkboxes use css variables.
